test/jdk/java/awt/event/MouseEvent/ResizeMouseExitEnterMisfire/ResizeMouseExitEnterMisfire.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.swing.SwingUtilities;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import java.awt.Point;
+import java.awt.Rectangle;
+import java.awt.Robot;
+import java.awt.event.InputEvent;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+/**
+ * @test
+ * @bug 8254841
+ * @summary Tests spurious firing of mouseEntered and mouseExit on resize
+ * @run main ResizeMouseExitEnterMisfire
```

Comment:
I guess the test should be marked headful.
Also, did you test on all platforms by giving CI job since you did not mark it only for mac?
It seems it fails on windows.

Reply:
I'm currently unable to test windows platform, so I'm not sure..

Reply:
Not a "CI" job (CI jobs run on the integrated fixes in mainline) but you can submit a test job (ask me how)

Yes this test absolutely needs
@key headful 
else it will cause problems in the CI jobs.